boxoffice/static/js/views/admin_discount_policy.js

```diff
@@ -25,6 +25,10 @@ export const DiscountPolicyView = {
           org: org_name,
           title: title,
           discount_policies: discount_policies,
+          paginated: paginated,
+          total_pages: total_pages,
+          current_page: current_page,
+          pages: Array.from(Array(total_pages).keys()).map(x => ++x),
```

Comment:
Making an array starting from `1` to `total_pages`. It's an one liner. But now I feel a simple for loop will be more readable. Reactive doesn't seem to have a way to run a loop for a given length, so have to send a predefined array.